src/applications/coronavirus-research/widgets/HeightWidget.jsx

```diff
@@ -13,18 +13,24 @@ function getEmptyState() {
     },
   };
 }
-
+function setLabelStyles(newState) {
+  let labelMarginLeftClass = 'vads-u-margin-left--1';
+  if (
+    newState &&
+    newState.touched.feet &&
+    newState.touched.inches &&
+    (!newState.value.feet || !newState.value.inches)
+  ) {
+    labelMarginLeftClass = 'vads-u-margin-left--3';
+  }
+  return `vads-l-col--1 vads-u-margin-top--3 ${labelMarginLeftClass}`;
+}
 const calculateHeight = ({ feet, inches }) =>
```

Comment:
is this intended to return a height in decimal value?

Reply:
@mchelen-gov  This does not and should not return a decimal value.  This translates the value of height in feet and inches to height in inches